core/sqf/src/seatrans/tm/hbasetmlib2/hbasetm.h

```diff
@@ -139,9 +139,9 @@ class CHbaseTM : public JavaObjectInterfaceTM
       JM_REGTRUNCABORT,
       JM_DROPTABLE,
       JM_RQREGINFO,
-      JM_LAST
+      JM_TMLAST
    };
-   JavaMethodInit JavaMethods_[JM_LAST];
+   JavaMethodInit JavaMethods_[JM_TMLAST];
```

Comment:
This is critical!

Reply:
There is  another define of JM_LAST in this file, and their value is different.